Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2755,6 +2781,8 @@ - (void)_didCompleteAnimatedResize
     [self _didStopDeferringGeometryUpdates];
 }
 
+// This should do at least all the things that had been skipped in `-[WKWebView _frameOrBoundsMayHaveChanged]`,
```

Comment:
If you're going to have this comment, I think it should at least be inside the function. And maybe it'a a reasonable comment? Just unusual.